pyproject.toml

```diff
@@ -124,17 +133,30 @@ description = "sync test data"
 cmd = "python b3d_pull.py -ow"
 cwd = "src/b3d/bucket_utils"
 
-[tool.pixi.feature.core.tasks.docs]
-description = "render docs to docs/static"
-cmd = "python -m pdoc --logo https://github.com/probcomp/b3d/assets/66085644/50bc2fc3-c9cd-4139-bed6-9c6d53933622 --math --mermaid -o docs/static/ b3d"
-env = { PDOC_ALLOW_EXEC = "1" }
-
 [tool.pixi.pypi-options]
 index-url = "https://pypi.org/simple"
 extra-index-urls = [
     "https://oauth2accesstoken@us-west1-python.pkg.dev/probcomp-caliban/probcomp/simple",
 ]
 
+[tool.pixi.feature.no-cuda]
```

Comment:
the `no-cuda` features just pulls in CPU-capable versions of `jaxlib`, `pytorch`, etc.